templates/default/cfnconfig.erb

```diff
@@ -1,22 +1,22 @@
-stack_name=<%= node['cluster']['stack_name'] %>
-preinstall=<%= node['cluster']['preinstall'] %>
-preinstall_args=(<%= node['cluster']['preinstall_args'] %>)
-postinstall=<%= node['cluster']['postinstall'] %>
-postinstall_args=(<%= node['cluster']['postinstall_args'] %>)
-region=<%= node['cluster']['region'] %>
-scheduler=<%= node['cluster']['scheduler'] %>
-scheduler_slots=<%= node['cluster']['scheduler_slots'] %>
-instance_slots=<%= node['cluster']['instance_slots'] %>
-ephemeral_dir=<%= node['cluster']['ephemeral_dir'] %>
-ebs_shared_dirs=<%= node['cluster']['ebs_shared_dirs'] %>
-proxy=<%= node['cluster']['proxy'] %>
-node_type=<%= node['cluster']['node_type'] %>
-cluster_user=<%= node['cluster']['cluster_user'] %>
+cfn_stack_name=<%= node['cluster']['stack_name'] %>
```

Comment:
This does not have `cfn_` prefix in PC2

Reply:
Fixed here: https://github.com/aws/aws-parallelcluster-cookbook/pull/1102